injected/src/content-scope-features.js

```diff
@@ -60,8 +63,7 @@ export async function init(args) {
     }
     registerMessageSecret(args.messageSecret);
     initStringExemptionLists(args);
-    const resolvedFeatures = await Promise.all(features);
```

Comment:
Was looking at this the other day. This used to be `features.push`'ing a promise I think.